skin.json

```diff
@@ -137,6 +137,10 @@
     "timeToShow": "10"
   },
   "controlBar": {
+    "volumeView": {
+      "progressColor": "#FFFFFFFF",
+      "thumbColor": "#FFFFFFFF"
```

Comment:
Believe it or not it IS on Android. But yeah, we can come up with something
better.

On Wed, Feb 17, 2016 at 11:16 AM, Dustin P. <notifications@github.com>
wrote:

> In skin.json
> <https://github.com/ooyala/skin-config/pull/112#discussion_r53215318>:
>
> > @@ -137,6 +137,10 @@
> >      "timeToShow": "10"
> >    },
> >    "controlBar": {
> > +    "volumeView": {
> > +      "progressColor": "#FFFFFFFF",
> > +      "thumbColor": "#FFFFFFFF"
>
> Is "thumb" the mobile terminology for the draggable element?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ooyala/skin-config/pull/112/files#r53215318>.
>